Libraries/Opc.Ua.Server/Subscription/IMonitoredItem.cs

```diff
@@ -94,6 +96,13 @@ public interface IMonitoredItem
         /// </summary>
         bool IsReadyToTrigger { get; set; }
 
+        /// <summary>
+        /// Gets or Sets a value indicating whether the item is in ResendData state
+        /// 0 means the MI is not in ResendData state
+        /// 1 means the MI is in ResendData state
+        /// </summary>
+        int IsResendData { get; set; }
+
```

Comment:
I think `SetResendData` or `RequestResendData` would better describe the intention of this property. Maybe it should be a method like SetTriggered()?